config/mainstream_content.json

```diff
@@ -0,0 +1,24 @@
+[
```

Comment:
Is this _all_ mainstream content? Or just stuff meant to have the old sidebar? If it's the latter, could we please have a more descriptive filename/folder structure to avoid confusion (eg `mainstream_content_with_curated_sidebar.json` or similar). It might also be nice to reflect that in the `MainstreamContentFetcher` class, which sounds like it's fetching all mainstream content (which it isn't).